### PR TITLE
Security Department Remap

### PR DESCRIPTION
**Pull request by Eyeveri**
feedback thread: https://forums.aurorastation.org/topic/18048-security-re-map

---

I decided to remap the entire Deck 2 security department. I personally did not like how it was designed around the communal area. It felt like a large waste of space, especially since communal is barely ever used (most antags die unfortunately). 
The Armory is now across from the HoS office, wardens office is still attached to the Brig but it is across from the isolation cell now. The equipment room has replaced the processing area. The bathroom has been shrunk tremendously, and is right behind the briefing room. (spawn). Process cells are across from the brig cells, and isolation is directly below it. 

It is important to note that while there is the same amount of cells, they are smaller. This is to encourage the use of communal more often. I have made it slightly easier for antagonists to escape communal as well, thus giving QOL to antags who are overwhelmed by Security. 

There is a small maintenance tunnel that connects both sides of security together now, it runs along where the old communal was. My three goals for this are:
- Bring the Armory closer to the HoS office, and central ring.. Similar to how it was on the Aurora. 
- Turn a brig sentence into a way to impact the round further, rather than a place to be stuck in indefinitely. I also hope this will encourage interaction with prisoners more.
- Utilize space to the best of my ability. I did my absolute best to make sure this remap maintains the same "compact" feel that being on a star-ship/cruiser would require.

While also ensuring QOL is not impacted negatively. My only concern is that when it comes to the stairway. If people clog up trying to get into the equipment room, I will just rearrange the doors. 

I want to try and make a "ready room" and attach it to the Armory but that will come at a later date. 

**screenshots**
-
<details>
<summary>click to view</summary>

armorylookscoolnow
BriefingAndToiler
CenterOfBrig
Communal
Equipment room
Isolation cells
MaintenanceArea
Warden Office and Communal entry

_(edited by SleepyGemmy @ 2023-02-25: added feedback thread link. updated PR description for readability.)_

### CODE REVIEW

**Comment by Eyeveri** [comment]
!review

**Comment by Eyeveri** [comment]
!map

**Comment by Eyeveri** [comment]
!check

**Comment by Eyeveri** [comment]
I did not touch an away site, I'm unsure as to why it fails sometimes or passes.

**Review by RyverStyx — DISMISSED** [review]

**Comment by Eyeveri** [comment]
!wip

**Comment by Eyeveri** [comment]
One more commit and It'll be ready for review again.

**Comment by Eyeveri** [comment]
!review

**Comment by Eyeveri** [comment]
!resolve

**Comment by Eyeveri** [comment]
why aren't the checks running? @SleepyGemmy

**Review by RyverStyx — APPROVED** [review]

File: aurorastation.dme [hunk 1]
```diff
@@ -10,6 +10,7 @@
 #define DEBUG
 // END_PREFERENCES
 // BEGIN_INCLUDE
+#include "_maps\sccv_horizon.dm"
```
Comment (alsoandanswer) [inline aurorastation.dme:13]: untick this please

**Review by alsoandanswer — APPROVED** [review]

File: aurorastation.dme (hunk 1 above)
Comment (Eyeveri) [reply aurorastation.dme]: done

**Pull request merged**